web/rainmaker/dev-packages/egov-ui-kit-dev/src/config/forms/specs/PropertyTaxPay/utils/reusableFields.js

```diff
@@ -431,16 +447,19 @@ export const beforeInitFormForPlot = {
 export const city = {
   city: {
     id: "city",
-    jsonPath: "Properties[0].address.city",
+    jsonPath: "PropertiesTemp[0].address.city",
```

Comment:
is that required to change to PropertiesTemp and while edit prefilling is happening?

Reply:
Prefilling is happening. This city is not being used anywhere. I will change this to original one.